guides/common/modules/proc_viewing-booted-container-images.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Viewing_booted_container_images_{context}"]
+= Viewing booted container images
+These actions streamline the management of image mode hosts, providing flexibility and control over host operations through {Project}.
+
+.Prerequisites
+* *Pull-Mode Remote Execution*: For pull-mode remote execution users, the `foreman_ygg_worker` package must be included in the image mode host's container image.
```

Comment:
This is relevant for REX jobs, right? So it should go together with the REX jobs, right?

Additionally, I think that the `foreman_ygg_worker` package by itself is insufficient and users also need `yggdrasil`. Please, check that with a SME.

Reply:
`foreman_ygg_worker` also pulls in `yggdrasil` and `foreman_ygg_migration`

Reply:
Side note -- it looks like yggdrasil on image mode hosts might be broken, I'm following up with Endeavour, but we might need a known issue until it's sorted.

Reply:
It was confirmed from several SMEs that it's `foreman_ygg_worker`.